website/assets/js/pages/try-fleet/register.page.js

```diff
@@ -0,0 +1,58 @@
+parasails.registerPage('register', {
+  //  ╦╔╗╔╦╔╦╗╦╔═╗╦    ╔═╗╔╦╗╔═╗╔╦╗╔═╗
+  //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
+  //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
+  data: {
+    formData: { /* … */ },
+    // For tracking client-side validation errors in our form.
+    // > Has property set to `true` for each invalid property in `formData`.
+    formErrors: { /* … */ },
+
+    // Form rules
+    formRules: {
+      emailAddress: {required: true, isEmail: true},
+      password: {
+        required: true,
+        minLength: 12,
+        // Custom password validation to ensure it contains at least one letter, one number, and one special character. TODO: full list of special characters
+        custom: (password)=>{
+          return !! password.match(/[\!\@\#\$\%\^\&\*]/) && password.match(/\d/) && password.match(/\w/);
+        }
+      },
+    },
+    // Syncing / loading state
+    syncing: false,
+    // Server error state
+    cloudError: '',
+  },
+
+  //  ╦  ╦╔═╗╔═╗╔═╗╦ ╦╔═╗╦  ╔═╗
+  //  ║  ║╠╣ ║╣ ║  ╚╦╝║  ║  ║╣
+  //  ╩═╝╩╚  ╚═╝╚═╝ ╩ ╚═╝╩═╝╚═╝
+  beforeMount: function() {
+    //…
+  },
+  mounted: async function() {
+    //…
+  },
+
+  //  ╦╔╗╔╔╦╗╔═╗╦═╗╔═╗╔═╗╔╦╗╦╔═╗╔╗╔╔═╗
+  //  ║║║║ ║ ║╣ ╠╦╝╠═╣║   ║ ║║ ║║║║╚═╗
+  //  ╩╝╚╝ ╩ ╚═╝╩╚═╩ ╩╚═╝ ╩ ╩╚═╝╝╚╝╚═╝
+  methods: {
+
+    // Using handle-submitting to add firstName, and lastName values to our formData before sending it to signup.js
+    handleSubmittingRegisterForm: async function(argins) {
+      argins.firstName = argins.emailAddress.split('@')[0];
+      argins.lastName = argins.emailAddress.split('@')[1];
+      argins.signupReason = 'Try Fleet Sandbox';
+      await Cloud.signup.with(argins);
```

Comment:
Based on how handleSubmitting functions are designed to work, this should be `return await Cloud.signup.with(argins);`   (it likely won't matter in this case, but it's a good idea to do this everywhere, so that you can get ahold of the server response data easily in your HTML when/if needed down the road, without having to come back in here.  Plus it's the convention)